src/controllers/cluster.rs

```diff
@@ -65,6 +66,10 @@ impl TemplateSources {
         cluster.status = None;
         cluster.meta_mut().managed_fields = None;
         cluster.meta_mut().resource_version = None;
+        cluster.metadata.annotations = Some(cluster.annotations().clone()
+            .into_iter()
+            .filter(|(key, _)| annotation_filter(key.as_ref()))
```

Comment:
```suggestion
        cluster.metadata.annotations = Some(cluster.annotations().iter()
            .filter(|(key, _)| annotation_filter(key.as_ref()))
```
I think this would compile (maybe not) and avoid clones. `IntoIter` typically clones content, but we only need to pass a ref.

Reply:
I don't seem to get this working without cloning it.